integreat_cms/cms/templates/generic_confirmation_dialog.html

```diff
@@ -14,7 +14,7 @@
         <div class="w-full p-4 rounded shadow">
             <h2 id="confirmation-title"></h2>
             <h3 id="confirmation-subject" class="text-center mt-4 mb-6 break-words"></h3>
-            <p id="confirmation-text" class="mt-4 mb-6">
+            <p id="confirmation-text" class="mt-4 mb-6" style="white-space: pre-line">
```

Comment:
```suggestion
            <p id="confirmation-text" class="mt-4 mb-6 whitespace-pre-line">
```
I'd prefer tailwind classes over inline css styles.